src/main/mainEvents.ts

```diff
@@ -0,0 +1,19 @@
+/**

```

Comment:
For some reason this file has CRLF line-endings. Please change to LF.

Reply:
Fixed (rebased to drop the CRLF from history).